packages/chrome/src/index.tsx

```diff
@@ -31,17 +39,17 @@ const runScript = script =>
     )
   })
 
-const mergeState = (state, next) => merge({}, state, next)
+const mergeState = (state: any, next: any) => merge({}, state, next)
```

Comment:
Is `state` and `next` `{ colorMode: string, theme: Theme }`? Just guessing tbh.

Reply:
I wasn't sure so that was why I left it open. I'll take a look to understand which type this is